src/MizarWidgets/SamplingWithFrameTrackInputWidget.cpp

```diff
@@ -57,6 +64,19 @@ void SamplingWithFrameTrackInputWidgetBase::OnFrameTrackSelectionChanged(int ind
   frame_track_scope_id_ = GetFrameTrackList()->itemData(index, kScopeIdRole).value<uint64_t>();
 }
 
+void SamplingWithFrameTrackInputWidgetBase::OnStartMsChanged(const QString& time_ms) {
+  if (time_ms.isEmpty()) {
+    start_relative_time_ns_ = 0;
+    return;
+  }
+  bool ok;
+  constexpr uint64_t kNsInMs = 1000;
+  start_relative_time_ns_ = static_cast<uint64_t>(time_ms.toInt(&ok)) * kNsInMs;
+  if (!ok) {
+    start_relative_time_ns_ = std::numeric_limits<int>::max() * kNsInMs;
```

Comment:
I think there should be a comment at the field that max<int>*1000 is representing an "invalid start"... Probably its worth introducing a constant.

Reply:
Let's just set the start_relative_time_ns_ to `MAX_UINT64` in case of a malformed input.